ckan/templates/package/resource_edit_base.html

```diff
@@ -37,5 +37,6 @@ <h1 class="hide-heading">{% block form_title %}{{ _('Edit resource') }}{% endblo
 
 {% block scripts %}
   {{ super() }}
-  {% resource 'vendor/fileupload' %}
+  {#% resource 'vendor/fileupload' %#}
```

Comment:
why leave the old tags as comments?

Reply:
I was sure that there will be some alterations before this PR merged into core, so I just left these lines as examples of migration from fanstatic to web assets. I'm going to remove them in next commit